components/Airdrop/TotalRewards/TotalRewards.tsx

```diff
@@ -0,0 +1,168 @@
+import clsx from 'clsx'
+import { Box } from 'components/OffsetBorder/Box'
+import { InfoChip } from '../InfoChip/InfoChip'
+import useClipboard from 'react-use-clipboard'
+import styles from './totalRewards.module.css'
+import ActivityCopy from 'components/icons/ActivityCopy'
+
+type Props = {
+  totalPoints: number | null
+  totalIron: number | null
+  airdropHash: string | null
+}
+
+export const titlesByPhase = {
+  pool_three: 'Pull Request Points',
+  pool_one: 'Phase 1 Points',
+  pool_two: 'Phase 2 Points',
+  pool_four: 'Phase 3 Points',
+}
+
+const longestPoolName = Object.values(titlesByPhase).sort(
+  (a, b) => b.length - a.length
+)[0]
```

Comment:
This is carryover from the other row items so that the equals sign lines up... 🥸